src/test/java/com/marizueva/laboratory/hw3/fluentpageobject/tests/SiteTestWithSoftAssertions.java

```diff
@@ -0,0 +1,205 @@
+package com.marizueva.laboratory.hw3.fluentpageobject.tests;
+
+import com.marizueva.laboratory.hw3.fluentpageobject.pages.HomePage;
+import com.marizueva.laboratory.hw3.fluentpageobject.pages.ServiceDifferentElementPage;
+import com.marizueva.laboratory.hw3.fluentpageobject.pages.ServicesSubCategories;
+import com.marizueva.laboratory.hw3.testdata.DataProviders;
+import com.marizueva.laboratory.hw3.utils.DriverManager;
+import com.marizueva.laboratory.hw3.utils.TestDataProperties;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+import java.util.Properties;
+
+public class SiteTestWithSoftAssertions {
+    WebDriver driver;
+    SoftAssert softAssert;
+    HomePage homePage;
+    Properties testDataProps = null;
+    String homePageTitle = "Home Page";
+    String loggedUser = "ROMAN IOVLEV";
+
+    private String userName; //= "Roman";
+    private String password;
+
+    @BeforeClass(alwaysRun = true)
+    public void beforeClass() {
+
+        testDataProps  = TestDataProperties.getTestDataProps();
+        userName = testDataProps.getProperty("username");
+        password = testDataProps.getProperty("password");
+        softAssert = new SoftAssert();
+
+    }
+
+    @BeforeMethod
+    public void setUp() {
+        driver = DriverManager.setUpDriver();
+    }
```

Comment:
Looks like this part repeated from the src/test/java/com/marizueva/laboratory/hw3/fluentpageobject/tests/SiteTestWithHardAssertions.java

I suppose that it could be extracted to the base class